source/jormungandr/tests/chaos_disruptions_tests.py

```diff
@@ -182,15 +183,81 @@ def test_disruption_on_journey(self):
+class TestChaosDisruptionsUpdate(ChaosDisruptionsFixture):
+    """
+    Note: it is done as a new fixture, to spawn a new kraken, in order not the get previous disruptions
+    """
+    def test_disruption(self):
+        """
+        test /disruptions and check that an update of a disruption is correctly done
+        """
+        self.wait_for_rabbitmq_cnx()
+        query = 'disruptions?datetime=20140101T000000&_current_datetime=20140101T000000'
+        response = self.query_region(query)
+
+        assert response['disruptions'][0]['network']['disruptions']
+        eq_(len(response['disruptions'][0]['network']['disruptions']), 1)
+
+        status = self.query_region('status')
+        last_loaded_data = get_not_null(status['status'], 'last_rt_data_loaded')
+
+        #we create a disruption on the network
+        self.send_chaos_disruption("test_disruption", "base_network", "network", message='message')
+
+        #we sleep a bit to let kraken reload the data
+        self.poll_until_reload(last_loaded_data)
+
+        response = self.query_region(query)
+
+        assert 'disruptions' in response
+        eq_(len(response['disruptions']), 1)
+        assert response['disruptions'][0]['network']['disruptions']
+        eq_(len(response['disruptions'][0]['network']['disruptions']), 2)
+        for disruption in response['disruptions'][0]['network']['disruptions']:
+            if disruption['uri'] == 'test_disruption':
+                eq_(disruption['messages'][0]['text'], 'message')
+
+        status = self.query_region('status')
+        last_loaded_data = get_not_null(status['status'], 'last_rt_data_loaded')
+
+        #we update the previous disruption
+        self.send_chaos_disruption("test_disruption", "base_network", "network", message='update')
+        self.poll_until_reload(last_loaded_data)
+
+        response = self.query_region(query)
+
+        assert 'disruptions' in response
+        eq_(len(response['disruptions']), 1)
+        assert response['disruptions'][0]['network']['disruptions']
+        eq_(len(response['disruptions'][0]['network']['disruptions']), 2)
+        for disruption in response['disruptions'][0]['network']['disruptions']:
+            if disruption['uri'] == 'test_disruption':
+                eq_(disruption['messages'][0]['text'], 'update')
+
+        status = self.query_region('status')
+        last_loaded_data = get_not_null(status['status'], 'last_rt_data_loaded')
+
+        #we delete the disruption
+        self.send_chaos_disruption("test_disruption", "base_network", "network", is_deleted=True)
+        self.poll_until_reload(last_loaded_data)
+
+        response = self.query_region(query)
+        assert response['disruptions'][0]['network']['disruptions']
+        eq_(len(response['disruptions'][0]['network']['disruptions']), 1)
+        for disruption in response['disruptions'][0]['network']['disruptions']:
+            if disruption['uri'] == 'test_disruption':
```

Comment:
you can just assert disruption['uri'] != 'test_disruption', 'this disruption must have been deleted' no ?